### DISCUSSION TITLE
why i can move cursor beyond text columns max length?

### DISCUSSION THREAD

**Discussion by erlangparasu** [LazyVim/LazyVim#5206] [Q&A] [answered]
i have 1 line text that have 10 chars length.
in normal mode i can move my cursor to lines 15.
why this happen? what vim options to toogle this behavior?

**Comment by folke** [comment, marked as answer]
You enabled `virtualedit`

**Reply by erlangparasu** [reply]
i tried `:set virtualedit=` but my cursor still able to move far

**Reply by erlangparasu** [reply]
oh ok it works now, but i need to switch to terminal buffer first than go to file buffer. now file buffer cursor back to normal position
